### PR TITLE
async/.await support

### PR DESCRIPTION
**Pull request by prostomarkeloff**
This PR changes:
- async/.await support with `tokio` and `reqwest`
- Version is changed to `0.20`
- `reqwest` is updated to `0.10`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.20.0] - 2020-05-20
+### Changed
+- Move to `reqwest` 0.10
+- Support of synchronous interface is removed
+- async/.await support has come
```
Comment (awaitlink) [inline CHANGELOG.md]:
```suggestion
## Unreleased
### Changed
- `async`/`.await` support has arrived! :tada:
- Upgraded `reqwest` to `0.10`.

### Removed
- Synchronous access has been removed in favor of `async`/`.await`.
```

When version is `<1.0.0`, minor version changes are treated as major ones. I don't think there's any need to bump to `0.20.0` just for the nice number. I will bump the version and add the date of release myself (so please undo changes to the version number in other places too), no need to rush :)

File: README.md [hunk 2]
```diff
@@ -21,33 +21,37 @@ Add the dependency to your project:
...
 ```
 
 Now you can take a look at `rvk`'s [API documentation][docs.rs/rvk] to learn more about the available functions.
 
 # Example
 
-To use this example, you will **also** need the [`serde_json`][crates.io/serde_json] crate to deserialize the API response:
+To use this example, you will **also** need the [`serde_json`][crates.io/serde_json] crate to deserialize the API response.
+
+If you want to use `tokio::main` proc macro, you need to install `tokio` crate.
```
Comment (awaitlink) [inline README.md]:
```suggestion
To use this example, you will **also** need the [`serde_json`][crates.io/serde_json] crate to deserialize the API response, and the [`tokio`](https://crates.io/tokio) crate for the `tokio::main` attribute proc macro.
```

**Comment by prostomarkeloff** [comment]
Also I think in 0.20 version we should replace `failure` to `thiserror` (because it's deprecated) and maybe add support of auto deserializing API responses into structures (because I think the way represented now is a bit complicated). I will work on it in the next PR.

**Review by awaitlink — CHANGES_REQUESTED** [review]
See comments.

**Review by awaitlink — APPROVED** [review]
LGTM

**Pull request merged**